client/src/main/resources/process-0.1.xsd

```diff
@@ -331,9 +338,35 @@
             <xs:enumeration value="oozie"/>
             <xs:enumeration value="pig"/>
             <xs:enumeration value="hive"/>
+            <xs:enumeration value="spark"/>
         </xs:restriction>
     </xs:simpleType>
 
+    <xs:complexType name="spark-attributes">
+        <xs:annotation>
+            <xs:documentation>
+                List of Spark attributes to run Spark application:
+                    master element is optional and indicates the url of the Spark Master. Ex: yarn-client, yarn-master, or local. This will override the Spark interface, if define in cluster entity.
+                    mode element if present indicates the mode of spark, where to run spark driver program. Ex: client,cluster.
+                    name element indicates the name of the spark application.
+                    application-type indicates type of the spark application java, python or spark-sql.
+                    class element if present, indicates the spark's application main class.
+                    jar element indicates a comma separated list of jars or python files. Also provide the jar or file directory path to workflow attribute path
```

Comment:
@pallavi-rao  If user want to run python spark application, then they must specify with <jar> file option. It is as per Oozie Spark xsd. Ideally I prefer to use <file> instead of <jar>.